frameworks/engineering/mobile.md

```diff
@@ -212,6 +215,7 @@ topics:
           - "Writes code that serves as a definitive example for new engineers"
           - "Identifies and fixes security vulnerabilities"
           - "Sought out for reviewing complex changes or commenting on larger scale proposals"
+          - "Builds non trivial complex flows that serve as an example of a high quality product"
```

Comment:
Would it be worth getting something around interaction design here? "high quality" could mean a lot of things, code quality, design etc?